src/molecules/AutoPasSimpleMolecule.cpp

```diff
@@ -20,7 +20,7 @@ AutoPasSimpleMolecule::AutoPasSimpleMolecule(unsigned long id, Component* compon
 	} else if (_component != component and component != nullptr) {
 		Log::global_log->debug() << "AutoPasSimpleMolecule can only handle one component" << std::endl;
 		_component = component;
-		// Simulation::exit(32);
+		// mardyn_exit(32);
```

Comment:
```suggestion
		mardyn_exit(32);
```
If we had this exit, our recent discussion would have been resolved quicker ;)

Reply:
Definitely 😅
I also changed the debug out to error to make it visible in normal runs

Reply:
I am not sure if your suggested change in `Simulation.cpp` won’t break our CI, since for SEQ we are just using the same decomposition as for parallel, aren‘t we?
But: The validation checks are only conducted for parallel runs.